opendbc/car/tests/routes.py

```diff
@@ -95,6 +95,8 @@ class CarTestRoute(NamedTuple):
   CarTestRoute("f29e2b57a55e7ad5|2021-03-24--20-52-38", HONDA.HONDA_ACCORD),  # hybrid, 2021 with new style HUD msgs
   CarTestRoute("1ad763dd22ef1a0e|2020-02-29--18-37-03", HONDA.HONDA_CRV_5G),
   CarTestRoute("0a96f86fcfe35964|2020-02-05--07-25-51", HONDA.HONDA_ODYSSEY),
+  # TODO: UPDATE THIS
+  CarTestRoute("c443ca9eee127f66|2022-08-22--19-25-29", HONDA.HONDA_ODYSSEY_BOSCH),
```

Comment:
Does need an update, that route is no longer available. Would be great if you can get one, but it's not an absolute blocker for dashcam merge.

Reply:
I'll update the user branch and ask for a new route

Reply:
As a 2022 Odyssey owner I can provide a route.
Should it be with any specific fork?

Reply:
@jyoung8607
@csouers